hoot-core/src/main/cpp/hoot/core/io/DataConverter.cpp

```diff
@@ -294,7 +294,7 @@ void DataConverter::_convertToOgr(const QStringList& inputs, const QString& outp
       convertOps.setProgress(
         Progress(
           ConfigOptions().getJobId(), JOB_SOURCE, Progress::JobState::Running,
-          (float)(currentTask - 1) / (float)numTasks, 1.0 / (float)numTasks));
+          (currentTask - 1) / numTasks, 1.0 / numTasks));
```

Comment:
All instances of calls to `Progress()` have a `1.0 / (float)numTasks` that should be changed to `1.0f / (float)numTasks)`